frontend/src/Student/pages/Submit.js

```diff
@@ -11,16 +11,14 @@ import OtherMediaSubmissionForm from '../containers/OtherMediaSubmissionForm'
 const Submit = (props) => (
   <Container>
     <Switch>
-      <Route exact path='/submit' component={SubmissionFormChooser} />
-      <Route exact path='/submit/photo' render={() => <PhotoSubmissionForm forShow={props.forShow} />} />
-      <Route exact path='/submit/video' render={() => <VideoSubmissionForm forShow={props.forShow} />} />
-      <Route exact path='/submit/other' render={() => <OtherMediaSubmissionForm forShow={props.forShow} />} />
+      <Route exact path='/submit/:id' component={SubmissionFormChooser} />
```

Comment:
Not a _huge_ fan of this URL format here.

Thoughts about doing:

`/submit?to=<public_id>`
`/submit/photo?to=<public_id>`
`/submit/video?to=<public_id>`
`/submit/other?to=<public_id>`

or 

`/submit?to=<public_id>`
`/submit?to=<public_id>&type=photo`
`/submit?to=<public_id>&type=video`
`/submit?to=<public_id>&type=other`

Without the `to` param or an invalid/non-authorized `public_id`, you'd be redirected to your dashboard.

See #169 for my thoughts on using a `public_id`. We could just use the show's id for now.

Reply:
Probably going with /submit?to=<public_id>
/submit/photo?to=<public_id>
/submit/video?to=<public_id>
/submit/other?to=<public_id>